sdk/communication/AzureCommunication/Source/Identifiers.swift

```diff
@@ -26,17 +26,11 @@
 
 import Foundation
 
-@objc public protocol CommunicationIdentifier: NSObjectProtocol {
-    func getId() -> String
-}
+@objc public protocol CommunicationIdentifier: NSObjectProtocol {  }
 
 @objcMembers public class CommunicationUser: NSObject, CommunicationIdentifier {
     public let identifier: String
```

Comment:
Can we rename the field to `id` so that it matches Java and C#? Same for the other types.

```suggestion
    public let id: String
```

Reply:
In Objective-C `id` is considered an actual type. This may raise some confusion.

Reply:
I didn't know, let's keep `identifier` then.

(Talked offline, just commenting here for transparency :-) )